### ISSUE TITLE
Multiple definitions of individual annotations

### ISSUE THREAD

**Issue by mmauger** [bastibe/annotate.el#16]
I'm not sure what the workflow was to cause this but I have multiple copies of the exact same annotation in annotations file for the same file.  For example:

```
("/scp:xyz-remhost:/users/mmauger/proj/abcdef/ghijklmnopqrstuvx.sql" (1430 1450 "Added criteria") (7866 7866 "xxxxx") (7866 7866 "xxxxx") (17916 17917 "Do we need to accommodate time portions") (17916 17917 "Do we need to accommodate time portions"))
```

(I cleaned up the previously reported Tramp text properties on the file name.)

**Comment by bastibe** [comment]
I'd be very interested in the workflow that caused this. Maybe I should switch to a data structure that does not allow multiple entries per key.

**Comment by mmauger** [comment]
It may just require that you filter the overlays before written out to filter duplicates (cl-delete-duplicates)

**Comment by bastibe** [comment]
Should be fixed in 0.3.4.
